BEST-PRACTICES.md

```diff
@@ -0,0 +1,49 @@
+# Best Practices
+
+We're glad you're considering contributing to SUSHI! Below are a few best practices that we recommend for all contributions.
+
+## Before contributing
+
+Before contributing a feature or a bugfix, we recommend creating a GitHub issue if one does not exist. This allows the community to provide feedback on why an issue may be occurring or provide additional insight into a suggested feature.
+
+If a GitHub issue already exists for what you are planning to contribute, we recommend commenting on the issue to indicate that you are working on an implementation to avoid duplication of work.
+
+## Coding practices
+
+We recommend the following coding practices for high quality contributions:
+
+- Make all changes in a personal [fork](https://help.github.com/articles/fork-a-repo/) of this repository.
+- Use descriptive commit messages.
+- Prefer self-explanatory code as much as possible, but provide helpful comments for complex expressions and code blocks.
+- Add unit tests for any new or changed functionality, and update any existing tests that are impacted by your changes.
+  - SUSHI uses [Jest](https://jestjs.io/) as a testing framework.
+  - To run the full test suite, run `npm test`. Ensure all tests are passing.
+- Follow the code style and conventions as enforced by the lint configuration and as evidenced by the existing code.
+  - SUSHI uses [ESLint](https://eslint.org/) for code linting.
+  - To run the linter on all code, run `npm run lint`. Ensure there are no issues reported.
```

Comment:
Can we also include instructions on how to fix the linting? E.g. `npm run lint:fix`

Reply:
Done in [db172dd](https://github.com/FHIR/sushi/pull/1474/commits/db172dd4017b281678ddefe358e874e493770c89). Bullet points were similarly tweaked.